pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "charm-lib-interface-kubernetes-cni"
+version = "0.0.0"
+authors = [
+  {name="George Kraft", email="george.kraft@canonical.com"},
+]
+description = "Charm library for kubernetes-cni interface"
+readme = "README.md"
+requires-python = ">=3.8"
+dependencies = [
+  "ops",
+]
```

Comment:
this dep isn't REALLY used though -- maybe it could be if the `interface_kubernetes_cni` module had some typing in it.